mat64/triangular.go

```diff
@@ -91,22 +91,23 @@ func (t TransposeTri) UntransposeTri() Triangular {
 	return t.Triangular
 }
 
-// NewTriDense constructs an n x n triangular matrix with the orientation
-// specified by kind.
-// If len(mat) == n * n, mat will be used to hold the underlying data, if
-// mat == nil, new data will be allocated, and will panic if neither of these
-// cases is true.
-// The underlying data representation is the same as that of a Dense matrix,
-// except the values of the entries in the opposite half are completely ignored.
-func NewTriDense(n int, kind matrix.TriKind, mat []float64) *TriDense {
+// NewTriDense creates a new Triangular matrix with n rows and columns. If
+// len(data) == n*n, data is used as the underlying slice (changes to the matrix
```

Comment:
"data is used as the backing slice. ... Changes to elements of the returned TriiDense will be reflected in data."